### PR TITLE
Add function

### PR DESCRIPTION
**Pull request by whilestevego**
We often have to described this. 🛩  👍  💯

### CODE REVIEW

File: add_function/solution.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// add function
+// a: a number
+// b: a number
+// returns the sum of a and b
```
Comment (whilestevego) [inline add_function/solution.js]: Excellent comments 🤓

File: add_function/solution.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+function add(a, b) {
+  return a + b;
+}
+
+// usage:
+
+add(10, 10); // 20
+add(0, 90); // 90
```
Comment (whilestevego) [inline add_function/solution.js]: What happens when you add 0 to 0?

File: add_function/solution.js (hunk 1 above)
Comment (whilestevego) [reply add_function/solution.js]: 👋

**Comment by whilestevego** [comment]
General comments!

**Comment by zigzter** [comment]
:thinking:

**Pull request merged**